data/scripts/network/request_resource_balance.lua

```diff
@@ -0,0 +1,19 @@
+local callbacks = {
+	[RESOURCE_BANK_BALANCE] = Player.getBankBalance,
+	[RESOURCE_GOLD_EQUIPPED] = Player.getMoney,
+}
+
+local handler = PacketHandler(0xED)
+
+function handler.onReceive(player, msg)
+	local resource = msg:getByte()
+	local callback = callbacks[resource]
+	if not callback then
+		-- print("[Warning] Unknown resource type " .. string.format("%x", resource))
```

Comment:
The warning is nice, would also be good to add a reference to the function (prefix with `[Warning - network/request_resource_balance.lua]`)